sdk/eventgrid/microsoft-azure-eventgrid/src/main/java/com/microsoft/azure/eventgrid/customization/EventGridSubscriber.java

```diff
@@ -51,12 +52,13 @@ public EventGridSubscriber() {
      */
     @Beta
     public void putCustomEventMapping(final String eventType, final Type eventDataType) {
-        if (eventType == null || eventType.isEmpty()) {
-            throw new IllegalArgumentException("eventType parameter is required and cannot be null or empty");
-        }
-        if (eventDataType == null) {
-            throw new IllegalArgumentException("eventDataType parameter is required and cannot be null");
+        Objects.requireNonNull(eventType, "'eventType' cannot be null.");
```

Comment:
I just noticed the package name this change exists in.  The packages that start with `microsoft-azure-*` are owned by the service team rather than us. We own the ones that are start with `azure-*` (ie. azure-core, azure-messaging-eventhubs). Consequently, we try not to change existing behaviour of their client libraries.

Could you please revert this?